### PR TITLE
src/doc/common/python3.inv: Update

### PR DESCRIPTION
**Pull request by mkoeppe**
### :books: Description

This is our static copy of the sphinx inventory for Python, used for cross-references to the Python standard library.

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.
- [ ] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by kwankyu** [comment]
The intention is to get the inventory for the latest stable python version (instead of the python version that happens to be used in the developer platform)?

**Comment by mkoeppe** [comment]
Yes, exactly

File: src/doc/common/update-python-inv.sh [hunk 1]
```diff
@@ -9,11 +9,11 @@
```
Comment (kwankyu) [inline src/doc/common/update-python-inv.sh:10]: How about being explicit here:
```
Here is how to update it for the latest stable Python version:
```
?

**Review by kwankyu — APPROVED** [review]
Thanks.

**Comment by mkoeppe** [comment]
Thank you!

**Pull request merged**